deps/include/jconfigint.h

```diff
@@ -0,0 +1,9 @@
+#ifndef INLINE
+#if defined(__GNUC__)
```

Comment:
Perhaps it would be best if we just made the `__GNUC__` branch the default else branch.